### ISSUE TITLE
Green logo stays visible on first run of the app

### ISSUE THREAD

**Issue by psstoyanov** [GDQuest/learn-gdscript#97]
**Describe the bug**
Opening the page for the first time, doesn't remove the green loading logo.
The user can't click on anything while the loading logo is on screen.

**To Reproduce**
Steps to reproduce the bug:
1. Go to https://gdquest.github.io/learn-gdscript/
2. Make sure the cache is cleared (Ctrl+Shift+R - varies between browsers)
3. Wait for the page to load
4. The green Godot logo will be visible over the first page preventing other interaction

**Expected behavior**
The loading logo doesn't obscure the contents.
The user doesn't have to reload the page to interact with the contents

**Screenshots**
Known state of the bug exhibiting itself (cleared cache/ first run):
![Screenshot_20220103_211623](https://user-images.githubusercontent.com/3979134/147981509-94da887f-53c2-4652-856c-a162ff35f833.png)

**Information about your device (please complete the following information):**
 - Operating System: 
 ```
➜ screenfetch
 ██████████████████  ████████     pak0stpc@fr0sthutpc
 ██████████████████  ████████     OS: Manjaro 21.2.1 Qonos
 ██████████████████  ████████     Kernel: x86_64 Linux 5.15.7-1-MANJARO
 ██████████████████  ████████     Uptime: 1d 10h 4m
 ████████            ████████     Packages: 1989
 ████████  ████████  ████████     Shell: zsh 5.8
 ████████  ████████  ████████     Resolution: 3440x1440
 ████████  ████████  ████████     DE: KDE 5.89.0 / Plasma 5.23.4
 ████████  ████████  ████████     WM: KWin
 ████████  ████████  ████████     GTK Theme: Breeze [GTK2/3]
 ████████  ████████  ████████     Icon Theme: breeze-dark
 ████████  ████████  ████████     Disk: 716G / 969G (75%)
 ████████  ████████  ████████     CPU: AMD Ryzen 7 5800X 8-Core @ 16x 3.8GHz
 ████████  ████████  ████████     GPU: AMD Radeon RX 6800 (SIENNA_CICHLID, DRM 3.42.0, 5.15.7-1-MANJARO, LLVM 13.0.0)
                                  RAM: 8195MiB / 32070MiB


 ```
 - Browser: Firefox `95.0.2-1`

**Additional context**
Not applicable

**Comment by NathanLovato** [comment]
Thanks, I can reproduce this in Brave. It's weird, it wasn't happening earlier today with this build. I'll send this to a teammate to check in priority

**Comment by NathanLovato** [comment]
Weird, I can't reproduce this in Firefox here. It only happens in Brave on my computer.

**Comment by NathanLovato** [comment]
Also, in Brave, it's inconsistent. Sometimes it disappears correctly, sometimes the logo stays visible.

**Comment by psstoyanov** [comment]
@NathanLovato , you are right. 
It is inconsistent - a bit like 50:50 to get it or not for me on Firefox with the cache cleared.

**Comment by NathanLovato** [comment]
Fixed by f81e22333775ef23f0e0fe24d349695c38e2580b

The issue was a race condition in loading events report - you could get a loading progress event after the done event sometimes on first load, causing the bar to stay visible after the loading's end.
